.travis.yml

```diff
@@ -19,7 +21,9 @@ jobs:
       script: diff -u <(echo -n) <(gofmt -d -s .)
     - name: "golint"
       install: go get -u golang.org/x/lint/golint
-      script: golint -set_exit_status
+      script: golint -set_exit_status ./...
+    - name: "govet"
+      script: go vet
```

Comment:
shouldn't go vet be also run as "go vet ./..."

Reply:
Yeah, you are right